assets/src/editor/events/tickets/editor-ticket/list-view/editor-ticket-entities-list-view.js

```diff
@@ -105,4 +106,15 @@ const EditorTicketEntitiesListView = ( {
 	);
 };
 
+EditorTicketEntitiesListView.propTypes = {
+	entities: PropTypes.array.isRequired,
+	displayTicketDate: PropTypes.string,
+	htmlClass: PropTypes.string,
+	doRefresh: PropTypes.func,
```

Comment:
can you plz remove this prop from this component and from `ticketsListTableRow` as well as from anywhere else you see it?

This was added by someone else but is not necessary.